test/ruby/CMakeLists.txt

```diff
@@ -43,3 +43,9 @@ set_property(TEST RubyLocalPlugins APPEND PROPERTY ENVIRONMENT
 # wanna test
 set_property(TEST RubyLocalPlugins APPEND PROPERTY ENVIRONMENT
     "TYPELIB_RUBY_PLUGIN_PATH=")
+
+add_custom_target(
+    rubocop rubocop
```

Comment:
Can you add rubocop os a test dependency?

Reply:
:+1: done